src/transformers/hf_api.py

```diff
@@ -64,6 +64,8 @@ def __init__(
         self.lastModified = lastModified
         self.size = size
         self.rfilename = rfilename
+        for k, v in kwargs.items():
+            setattr(self, k, v)
```

Comment:
(out of scope). Do we need S3Obj and S3Object?
Can we call it HFAPI or something more predictable than the current `HfApi`

Reply:
Yes the objects are slightly different for mostly historical reasons on the server (upload, vs. list of all public models)